### ISSUE TITLE
 [Discussion] Move patches to a GitHub repository

### ISSUE THREAD

**Issue by illusion0001** [RPCS3/rpcs3#11538]
Would appreciate opinions from people who actually write patches.

Reasons for why patches on a GitHub repository are:

- Wiki doesn't have proper version control (no git, no pull requests)
  Revisions/changes can be unrelated to patches.
- Can't be opened in a preferred editor (i.e. VSCode)
- Can't be easily edit multiple patches at once. (i.e. capitalize all of them for consistency)
- Patches cannot be downloaded if compatibility server is down.
- Adding new patches to non existent game pages are unnecessary hassle
- The original proposal was for patches to have a repository, not be hosted from the wiki: https://github.com/RPCS3/rpcs3/issues/8366#issuecomment-64103946

**Comment by Margen67** [comment]
If you want examples of game patches, etc. hosted on GitHub:
  * https://github.com/ActualMandM/cemu_graphic_packs
  * https://github.com/xenia-canary/game-patches

**Comment by Minacat1** [comment]
![wheel-of-fortune-answers-and-solutions](https://user-images.githubusercontent.com/93951815/157352355-de75cd77-ec2c-40be-b202-09c7e960306d.jpg)

**Comment by marcoluc97** [comment]
Thanks @illusion0001 for take my initial proposal as example and yeah I was talking about a database like, similar to PCSX2 GameDB or like precedent posts already said. Hoping to see RPCS3 team interest in this implementation idea.

**Comment by illusion0001** [comment]
There was a typo that was difficult to spot on the wiki because unrelated changes were mixed in, maybe they'll consider moving to git or smth. But there's always a question of who will be approved to have write access to the repository (like automod) on wiki to approve changes

**Comment by marcoluc97** [comment]
Yeah that's something that can happen I'd there is no validation or control over push change. So I also vote for move to a better managed situation, and like I said there are examples to take as templates instead of creating it from zero.

**Comment by Ninetime** [comment]
Given that the patches already pull from the wiki, are there any other concerns or bottlenecks that would prevent them being moved to a git repo?
